src/components/footer/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import classNames from './footer.module.scss';
+
+const Footer = () => {
+  return (
+    <footer className={classNames.infoRepo}>
+      <p>
+        The contents of this website are deployed from this{' '}
+        <a
+          href="https://github.com/Real-Dev-Squad/website-members"

```

Comment:
Can we change this link and put `Status-sites` GitHub Repo link?

https://github.com/Real-Dev-Squad/website-status

Reply:
@whyDontI "status-site" link has been updated. Please check.